UnityMixedCallstack.csproj

```diff
@@ -1,14 +1,16 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">11.0</VisualStudioVersion>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
     <!-- <VSSDK140Install Condition="'$(VSSDK140Install)' == ''">$(MSBuildProgramFiles32)\Microsoft Visual Studio 14.0\VSSDK\</VSSDK140Install> -->
-    <VSSDK170Install Condition="'$(VSSDK170Install)' == ''">$(MSBuildProgramFiles32)\Microsoft Visual Studio\2019\Professional\VSSDK\</VSSDK170Install>
+    <VSSdkPath Condition="'$(VisualStudioVersion.Contains(`17`))'">$(ProgramFiles)\Microsoft Visual Studio\2022\VSSDK\</VSSdkPath>
```

Comment:
For me this is in: `C:\Program Files\Microsoft Visual Studio\2022\Professional\VSSDK` -- what version of VS are using?

Reply:
I installed 2022 Professional to a custom location. (just removed "Professional" from the default path). We can set it back to default path. Couldn't find a registry or other way to get this info (now that vswhere is needed to locate VS installations.)

Reply:
Yeah I also briefly ran down the rabbit hole of attempting to automagically determine install location without much success. If you could change this to the default path that would be great! Once this lands I'll upload a visix installer for VS2022 as a release here.

Reply:
Changed to default paths in https://github.com/Unity-Technologies/UnityMixedCallstack/pull/14/commits/458bcd4ebae0f495d0fc0328773cacca20ab1577